samples/apps/copilot-chat-app/webapi/CopilotChat/Options/PromptsOptions.cs

```diff
@@ -131,7 +131,7 @@ public class PromptsOptions
         this.SystemChatContinuation
     };
 
-    internal string SystemChatPrompt => string.Join("\n", this.SystemChatPromptComponents);
+    internal string SystemChatPrompt => string.Join("\n\n", this.SystemChatPromptComponents);
```

Comment:
intentional?

Reply:
Yes, this is to insert a blank line between each component.

Reply:
Why is this necessary? Could you add a comment?